site/src/components/SignupForm/signupform.style.js

```diff
@@ -22,15 +22,27 @@ const SignupFormWrapper = styled.section`
 
 	.desc-text {
 		margin: 1rem;
-		p{
+		width: 50%;
+		text-align: justify;
+		p:first-child{
 			font-size: 1.35rem;
+			margin-bottom: 3rem;
 		}
 
 		@media screen and (max-width: 1024px) {
-			img, div {
-				display: none;
+			div {
+				displya: block;
 			}
 			margin: 1rem 1rem 2rem 1rem;
+			width: 100%;
+			text-align: start;
+		}
+	}
+
+	.desc-callout{
+		@media screen and (max-width: 1024px){
+			position: relative;
+			top: 930px;
```

Comment:
Use rem, it would come around `58rem` or so.

Reply:
Also, you need to update the width, and margin here. It is being rendered in full-width here which is not desired.
```
width: fit-content;
margin: auto
```

Reply:
The reason I said, `58rem` was otherwise, there would be a lof of whitespace beneath the form. And, that's the case now with `62rem`. Update to 58rem, it is fine.